src/resolvers.test.js

```diff
@@ -0,0 +1,42 @@
+const { resolvers } = require('./resolvers');
+
+const awsSecretManagerResolver = resolvers.asm;
+
+describe('AWS Resolver', () => {
+  let consoleOutput = [];
+  const mockedWarn = output => consoleOutput.push(output);
```

Comment:
🤔 Thoughts on just validating input and output?  I'd rather not enforce this tight of a constraint on functionality, because changing the error/warning message seems totally an ok thing to do that shouldn't cause tests to fail (i.e. personally I would just get rid of the snapshot matches).

Aside from that, the one test for a recursive lookup, and the nits, this looks great.

Reply:
I'm fine with that. 👍